py3status/modules/window_title.py

```diff
@@ -42,20 +45,35 @@ class Py3status:
     format = '{title}'
     max_width = 120
 
+    def post_config_hook(self):
+        self.empty_properites = {
+            x: '' for x in self.py3.get_placeholders_list(self.format)}
+        self.empty_properites.update({'title': ''})
```

Comment:
this should already be in the dict

I think it is because you have `{'{title}': '', ...}` hence the `[1:-1]`

also `properties` is a better spelling.  but maybe `placeholder_defaults` sounds better?

a `# we need empty defaults for when we have no data` or better is always good

Reply:
>this should already be in the dict

If @Artlant wanted only `{class}`, then it could trip on empty windows because of existing `max_width` config. `get()` both should have `title`. 

Will add comment.